pipenv/vendor/tomlkit/container.py

```diff
@@ -189,7 +189,12 @@ def remove(self, key):  # type: (Union[Key, str]) -> Container
         if idx is None:
             raise NonExistentKey(key)
 
-        self._body[idx] = (None, Null())
+        old_data = self._body[idx][1]
+        trivia = getattr(old_data, "trivia", None)
+        if trivia and getattr(trivia, "comment", None):
+            self._body[idx] = (None, trivia)
```

Comment:
The trivia contains string, just comment only please

Reply:
ah ok